src/java.base/share/classes/java/time/temporal/ChronoField.java

```diff
@@ -586,8 +586,11 @@ public enum ChronoField implements TemporalField {
      * <p>
      * This field is strictly defined to have the same meaning in all calendar systems.
      * This is necessary to ensure interoperation between calendars.
+     * <p>
+     * Range of {@code InstantSeconds} is between {@link Instant#MIN} and {@link Instant#MAX}
```

Comment:
Nit: `InstantSeconds` -> `INSTANT_SECONDS`

Reply:
Hello Naoto, I had used `InstantSeconds` to keep it consistent with how a similar doc is used for the `EPOCH_DAY` field. Let me know if you still prefer this to be `INSTANT_SECONDS` and I will update it.

Reply:
With the @code tag, I initially thought it can be used programmatically, but apparently, it was simply an Enum name. Users may not notice it till they see the source (or run its `toString()`). I am fine with either way, but probably keep consistent with `EPOCH_DAY` where it is not using @code tag.

Reply:
Hello Naoto, that's good point. I've updated the PR to remove `{@code}`.